presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -72,7 +72,7 @@ public static Slice parseLine(@SqlType(StandardTypes.VARCHAR) Slice input)
     @SqlType(GEOMETRY_TYPE_NAME)
     public static Slice stPoint(@SqlType(StandardTypes.DOUBLE) double x, @SqlType(StandardTypes.DOUBLE) double y)
     {
-        OGCGeometry geometry = createFromEsriGeometry(new Point(x, y), null);
+        OGCGeometry geometry = createFromEsriGeometry(new Point(x, y), SpatialReference.create(4326));
```

Comment:
What's this constant value? Please make it a static final and give a proper name to it.